package.json

```diff
@@ -437,6 +437,28 @@
             "markdownDescription": "Arguments to pass to SourceKit-LSP. Keys and values should be provided as individual entries in the list. e.g. `['--log-level', 'debug']`",
             "order": 2
           },
+          "swift.sourcekit-lsp.backgroundIndexing": {
+            "type": "boolean",
+            "default": false,
+            "markdownDescription": "Enable or disable background indexing. This option has no effect in Swift versions prior to 6.0.",
```

Comment:
I would like to mark this as an experimental option. Maybe by prefixing it with `Experimental:`.